### ISSUE TITLE
Minibatch raises indexing deprecation warning

### ISSUE THREAD

**Issue by OriolAbril** [pymc-devs/pymc#4664]
## Description of your problem
Using `pm.Minibatch` still raises 

```
FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
```
which were common before taking over Theano and updating its code, this one seems to linger still. Seen in https://github.com/pymc-devs/pymc-examples/pull/98

Source: https://github.com/pymc-devs/pymc3/blob/v4/pymc3/data.py#L320

**Comment by danhphan** [comment]
Hi @OriolAbril and @michaelosthege If this issues has not fixed yet, I'm happy to work to this task :)

**Comment by michaelosthege** [comment]
Hi @danhphan , that'd be awesome! The warning is still there. You can run `pytest -sv pymc\tests\test_minibatches.py` to confirm.

**Comment by danhphan** [comment]
Hi @michaelosthege, this issue was fixed in #5513, so I think it can be closed. Thank you!
